docs/elements/hx-busy/index.html

```diff
@@ -3,6 +3,7 @@
 minver: 0.4.0
 also:
   components/loaders: Loaders
+  guides/react-compatibility: React Compatability
```

Comment:
Link to the "Significant White Space" section of the React Compatibility guide (like in the Badges component).

Reply:
fixed